tests/baselines/reference/genericConditionalConstrainedToUnknownNotAssignableToConcreteObject.errors.txt

```diff
@@ -9,7 +9,7 @@ tests/cases/compiler/genericConditionalConstrainedToUnknownNotAssignableToConcre
                 Type 'unknown' is not assignable to type 'A'.
                   Type 'ReturnType<FunctionsObj<T>[string]>' is not assignable to type 'A'.
                     Type 'unknown' is not assignable to type 'A'.
-                      Property 'x' is missing in type '{}' but required in type 'A'.
+                      Type 'unknown' is not assignable to type 'A'.

```

Comment:
This looks just like what I got when I changed the apparent type of `unknown` without strictNullChecks, if that's any indication of something missing: https://github.com/microsoft/TypeScript/pull/47517#discussion_r788228238

Reply:
Because I did, but only sometimes. There's two `unknown` flavors with this commit, one with an object apparent type and one without - right now, I only use the objecty one for loose mode type parameter constraints. Could I also use it here? Sure. Do I need to? Probably not.

Reply:
I don't know all of the details, I was just looking at how the message was duplicated in hopes you might know how to make it not show up twice (I wasn't able to figure it out at the time, but this was a while ago)

Reply:
I think it's more than just duplicated once, it's also printed again 3 lines higher - I think in this case we're just printing multiple branches of error results rather than just one, this one is just the only one that gets the apparent type.

Reply:
Yeah. At the end of the day, the overlap between people who are actually type checking and also have strictNullChecks disabled seems... small and weird, so I don't personally care so much about goofy errors. Even VS Code's default project defaults to structNullChecks=true.